NewBlackAuthenticator/bin/Debug/netcoreapp3.1/NewBlackAuthenticator.runtimeconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "runtimeOptions": {
+    "tfm": "netcoreapp3.1",
```

Comment:
You don't HAVE to do this, but just know we're running .net 6.0, you've built against 3.1 😉